README.md

```diff
@@ -48,7 +48,6 @@ modules: [
 		config: {
 			updateInterval: 10000,
 			animationSpeed: 0,
-			align: 'right', // align labels
```

Comment:
in my repo, I don't remove that line. I wonder myself, why github assumed it is removed. I'm retrying re-submit my repo.